tank_lab_to_nwb/convert_towers_task/convert_towers.py

```diff
@@ -8,39 +8,37 @@
 
 n_jobs = 1  # number of parallel streams to run
 
-base_path = "D:/Neuropixels/TowersTask"
+base_path = "D:/Neuropixels"
 # base_path = "/mnt/scrap/cbaker239/Neuropixels/TowersTask"
 
 # Manual list of selected sessions that cause problems with the general functionality
 exlude_sessions = []
 
-session_strings = ["PoissonBlocksReboot_cohort1_VRTrain6_E75_T_20181105"]
+virmen_session_strings = ["TowersTask/PoissonBlocksReboot_cohort1_VRTrain6_E75_T_20181105"]
+spikeglx_session_strings = ["Neuropixels/A256_bank1_2020_09_30/"
+                            "A256_bank1_2020_09_30_g0/A256_bank1_2020_09_30_g0_t0"]
 nwbfile_paths = []
-for j, session in enumerate(session_strings):
-    session_strings[j] = os.path.join(base_path, session_strings[j])
-    nwbfile_paths.append(os.path.join(base_path, session) + "_local_stub.nwb")
+for j, (virmen_session, spikeglx_session) in enumerate(zip(virmen_session_strings, spikeglx_session_strings)):
+    virmen_session_strings[j] = os.path.join(base_path, virmen_session_strings[j])
+    spikeglx_session_strings[j] = os.path.join(base_path, spikeglx_session_strings[j])
+    nwbfile_paths.append(os.path.join(base_path, virmen_session) + "_local_stub.nwb")  # name defaults to virmen session
```

Comment:
Instead of this, could we use a generator function that yields path virmen and spikeglx files? 
If we did that, at the bottom in `Parallel()` there would be no need to iterate over the lists again, we could just simply call the generator function.

Reply:
It seems that the client is more interested in the processing pipeline anyway, so most of this script (which is designed to run batches on processed session data) will not be used, so I'd focus more on just making sure the positional interface is pulling everything necessary on the prototype files.